src/Compilers/Core/Portable/Symbols/ManagedKind.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// PROTOTYPE
```

Comment:
It's not really clear what's PROTOTYPE about this. every place you have a prototype comment, I'd leave a little extra explaining what's incorrect or missing.

Reply:
Good idea--I've been using it a bit interchangeably for TODO. The purpose here is to just to make sure I document the new type and the property on TypeSymbol.